docs/02_cleos/02_how-to-guides/how-to-submit-a-transaction.md

```diff
@@ -1,18 +1,35 @@
-## Goal
+## Overview
 
-Push a transaction
+This how-to guide provides instructions on how to submit, or push, a transaction using the `cleos` CLI tool.
 
 ## Before you begin
 
 * Install the currently supported version of `cleos`
 
 * Understand the following:
-  * What is a transaction
-  * How to generate a valid transaction JSON
+  * What is a [transaction](https://developers.eos.io/welcome/latest/glossary/index/#transaction).
+  * How to generate a valid transaction JSON.
+    * Consult [cleos push transaction reference](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/push/push-transaction), and pay attention to option `-d` and `-j`.
+    * Consult [push transaction operation](https://developers.eos.io/manuals/eos/latest/nodeos/plugins/chain_api_plugin/api-reference/index#operation/push_transactions) for chain api plug-in, and pay attention to the payload definition.
 
-## Steps
+## Command Reference
 
-* Create a JSON snippet contains a valid transaction such as the following:
+See the following reference guides for command line usage and related options for the `cleos` command:
+
+* The [`push transaction` cleos sub-command](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/push/push-transaction).
+
+## Procedure
+
+The following steps show how to:
+
+1. [Create the transaction as JSON snippet.](#1-create-the-transaction-as-json-snippet)
+2. [Send the transaction.](#2-send-the-transaction)
```

Comment:
Good usage of numbered bullets.

Reply:
I am not sure about this one, I've commented in the previous one as well.